### PR TITLE
QoL tweaks (T-screenshake, recruitment speechshift, shovels 4 miners/masons, misc)

### PR DESCRIPTION
**Pull request by Red-Honk**
## About The Pull Request

[Quality of life]:

Screen shake toggle.
credit: LongSleeves
PR: https://github.com/Tree445/Hearthstone/pull/87

Adds shovel to miners: (Also gave it to the mason, because why not?)
credit: Dorodomki
PR: https://github.com/Darkrp-community/OpenKeep/pull/286

Fiddled with job recruitment spells, removed references to Roguetown in the priest section.

Removes steward forced flaw: (Let's face it, forced flaws limit roleplay depictions of the character in question)
credit: SaintBurgher
PR: https://github.com/Darkrp-community/OpenKeep/pull/754

Pointless discord verb hidden: (It doesn't work, it was never meant to work, it's just a TG leftover)
credit: BadAtThisGame302
PR: https://github.com/Darkrp-community/OpenKeep/pull/194

## Why It's Good For The Game

Quality of life is always good, immersion is nice, depiction of a character is great, hence why recruitment speeches were changed in case they don't say, I or the like.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/nobility/lady.dm [hunk 1]
```diff
@@ -72,6 +72,6 @@
 	new_role = "Servant"
 	recruitment_faction = "Servants"
 	recruitment_message = "Serve the crown, %RECRUIT!"
-	accept_message = "FOR THE CROWN!"
-	refuse_message = "I refuse."
+	accept_message = "FOR SERVITUDE!"
```
Comment (Cyprex) [inline code/modules/jobs/job_types/roguetown/nobility/lady.dm:75]: What?

File: code/modules/jobs/job_types/roguetown/nobility/lady.dm [hunk 2]
```diff
@@ -72,6 +72,6 @@
...
+	refuse_message = "decline."
```
Comment (Cyprex) [inline code/modules/jobs/job_types/roguetown/nobility/lady.dm:76]:
```suggestion
	refuse_message = "Decline."
```

File: code/modules/client/preferences_toggles.dm [hunk 3]
```diff
@@ -29,6 +29,14 @@
 		prefs.toggles ^= TOGGLE_FULLSCREEN
 		toggle_fullscreeny(prefs.toggles & TOGGLE_FULLSCREEN)
 
+/client/verb/toggle_screenshake()
+	set name = "Toggle Screen Shake"
+	set category = "Options"
+	set desc = ""
+	if(prefs)
+		prefs.shake = !prefs.shake
+		to_chat(src, "Screen shake toggled")
```
Comment (Cyprex) [inline code/modules/client/preferences_toggles.dm:38]:
```suggestion
		if(prefs.shake)
			to_chat(src, "Screen shake enabled")
		else
			to_chat(scr, "Screen shake disabled")
```
Please test this if you do commit it directly from github!

File: code/modules/jobs/job_types/roguetown/nobility/lady.dm (hunk 1 above)
Comment (katsukai) [reply code/modules/jobs/job_types/roguetown/nobility/lady.dm:75]: Yeah, I don't like this change.

File: code/modules/client/preferences_toggles.dm (hunk 3 above)
Comment (Red-Honk) [reply code/modules/client/preferences_toggles.dm:38]: Appreciated, I didn't really know how code that.

**Comment by Red-Honk** [comment]
> the recruitment spells new `accept_message` make no sense but the rest can be merged.

Changed it to "FOR SOVEREIGNTY!" I am trying to avoid the word I or the in accepts and refusals of requests, as to make it more neutral in case the character themselves never use those words, if that makes any sense. (You know, kinda breaks immersion when a character that never uses the or I says it when they get recruited, also, I refuse sounded too defiant for my liking.)

File: code/modules/jobs/job_types/roguetown/nobility/lady.dm (hunk 1 above)
Comment (Red-Honk) [reply code/modules/jobs/job_types/roguetown/nobility/lady.dm]: Fine, I changed it to "FOR SOVEREIGNTY!"

**Pull request merged**